docs/_docs/servers/contentservers/nntp.md

```diff
@@ -40,10 +41,20 @@ An example of generating your own cert/key pair:
 openssl req -newkey rsa:2048 -nodes -keyout ./config/nntps_key.pem -x509 -days 3050 -out ./config/nntps_cert.pem
 ```
 
-### Example Configuration
+## Write Access
+Authenticated users may write messages to a group given the following are true:
+
+1. They are connected security (NNTPS). This is a strict requirement due to how NNTP authenticates in plain-text otherwise.
+2. The authenticated user has write [ACS](../../configuration/acs.md) to the target message conference and area.
+
+> :warning: Not all [ACS](../../configuration/acs.md) checks can be made over NNTP. Any ACS requiring a "client" will return false (fail), such as `LC` ("is local?").
+
+## Example Configuration
 ```hjson
 contentServers: {
     nntp: {
+        allowPosting: true
+
         publicMessageConferences: {
```

Comment:
BTW: I will be changing `publicMessageConferences` soon to match the new style introduced with Gopher which allows wildcards & is easier to manage.